_posts/blog/2016-12-22-dwh-part1_getting_started.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: "A journey towards a custom data warehouse solution Part 1 : Getting Started"
+description:
+modified:
+categories: blog
+author: tino_noack
+excerpt: This is part one of a series of articles in which we would like to share our journey with you and our experiences towards a custom data warehouse solution. Part one starts with a brief introduction to upday including the kinds of data we work with and a description of our working environment. Furthermore, we come up with a tentative definition of a data warehouse (DWH), describe components regarding our use case, and identify tools which can be applied for implementation. Finally, we introduce a data lake as a more 'Big-Data-friendly' but provisional solution.
+tags: [Data, DWH, Data Warehouse, Big Data, Data Platform, Data Lake]
+image:
+date: 2016-12-22T00:00:00-04:00
+---
+# Abstract
+
+This is part one of a series of articles in which we would like to share our journey with you and our experiences towards a custom data warehouse solution. Part one starts with a
+brief introduction to upday including the kinds of data we work with and a description of our working environment. Furthermore, we come up with a tentative definition of a data
```

Comment:
there's one thing that's missing here: why? why should the reader care? 
If I get it right (please correct me, if I'm wrong), we had a problem: "having to cope with a huge volume of data from a vast number of sources". 
The DWH is upday's solution to this. It's our way of solving this "coping" problem. 

So instead of "Getting Started" - you're rather saying what our problem was.